tests/apis.js

```diff
@@ -52,80 +51,300 @@ test.cb.after.always(t => {
+  const random = Math.round(Math.random() * live.length - 1);
+  let expected = live[random];
+  if (expected === undefined) {
+    expected = cloneDeep(live[(live.length - 1)]);
+  }
+
+  Object.keys(expected.attributes).forEach(attr => {
+    if (attr.split('-').indexOf('referencer') > -1) {
+      delete expected.attributes[attr];
     }
   });
+
+  const model = allTypes.find(typ => {
+    return typ.id === expected['type-slug'];
+  });
+
+  let modelattrs = cloneDeep(model.attributes);
+  modelattrs = modelattrs.map(attr => {
+    if (attr.id.split('-').indexOf('referencer') < 0) {
+      return attr;
+    }
+
+    return false;
+  }).filter(attr => {
+    return attr !== false;
+  });
+
+  // because depth is checked during attributes, depth 1 and 0 are the same for the attributes function
+  const query = {
+    depth: 0,
+  };
+
+  const attributes = apiUtils.attributes(expected.attributes, modelattrs, allTypes, query);
+
+  return attributes.then(result => {
+    t.is(typeof result, 'object', 'Should contain result, an object.');
+    t.is(Object.keys(result).length, 2, 'Should contain two main objects.');
+  })
+  .catch(e => {
+    console.log(e); // eslint-disable-line no-console
+  });
 });
 
 //////////////////////////////
 // Utils - format
 //////////////////////////////
-test('Utils: Format Results - List', t => {
-  const formatted = apiUtils.format(content.slice(0, 9));
-
-  formatted.forEach(item => {
-    t.true(item.hasOwnProperty('id'), 'Contains ID');
-    t.true(item.hasOwnProperty('type'), 'Contains Type');
-    t.true(item.hasOwnProperty('type_slug'), 'Contains Type Slug');
-    t.true(item.hasOwnProperty('key'), 'Contains Key');
-    t.true(item.hasOwnProperty('key_slug'), 'Contains Key Slug');
-    t.true(item.hasOwnProperty('meta'), 'Contains Meta');
-    t.false(item.hasOwnProperty('attributes'), 'Does not contain attributes');
-    t.is(item.meta.url, `/api/types/${item.type_slug}/${item.id}`, 'URL points to full content item');
+test.serial.skip('Utils: Format Results - List', t => {
+  const formatted = apiUtils.format(live.slice(0, 9));
+
+  return formatted.then(result => {
+    result.forEach(item => {
+      t.true(item.hasOwnProperty('id'), 'Contains ID');
+      t.true(item.hasOwnProperty('type'), 'Contains Type');
+      t.true(item.hasOwnProperty('type_slug'), 'Contains Type Slug');
+      t.true(item.hasOwnProperty('key'), 'Contains Key');
+      t.true(item.hasOwnProperty('key_slug'), 'Contains Key Slug');
+      t.true(item.hasOwnProperty('meta'), 'Contains Meta');
+      t.false(item.hasOwnProperty('attributes'), 'Does not contain attributes');
+      t.is(item.meta.url, `/api/types/${item.type_slug}/${item.id}`, 'URL points to full content item');
+    });
   });
 });
 
-test('Utils: Format Results - Attributes', t => {
-  const item = Math.round(Math.random() * (content.length - 1));
-  let expected = cloneDeep(content[item]);
+test.serial.skip('Utils: Format Results - Attributes', t => {
```

Comment:
Should have a test for when there is no depth query and when there is a depth query